vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/RequiredComboboxStartupIT.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2000-2019 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.combobox.test;
+
+import com.vaadin.flow.component.combobox.testbench.ComboBoxElement;
+import com.vaadin.flow.testutil.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.TimeoutException;
+
+@TestPath("required-combobox-startup")
+public class RequiredComboboxStartupIT extends AbstractComponentIT {
+
+    @Test(expected = TimeoutException.class)
+    public void serverSideValidation_persistsOnBlur() {
+        open();
+
+        ComboBoxElement comboBox = $(ComboBoxElement.class).first();
+
+        Assert.assertEquals(Boolean.TRUE.toString(),
+                comboBox.getAttribute("invalid"));
+
+        TestBenchElement error = comboBox.$("vaadin-text-field").first()
+                .$(TestBenchElement.class).id("vaadin-text-field-error-0");
+
+        Assert.assertTrue(error.getSize().getHeight() > 0);
+        Assert.assertEquals("Must be false", error.getText());
+
+        // This should timeout and the validation error preserved on screen.
+        waitUntil(input -> error.getSize().getHeight() == 0);
```

Comment:
Instead of having `expected = TimeoutException.class`, wrap this line inside try-catch and add the `Assert.fail` inside the try-block.

This ensures that the test doesn't turn into a false-positive if some other part of the code throws `TimeoutException` before reaching this part.